### PR TITLE
Branch pangrwa storage

### PR DESCRIPTION
**Pull request by pangrwa**
* Edit Client to contain UniquePolicyList
* Edit JsonAdaptedClient to store JsonAdaptedPolicy
* Create JsonAdaptedPolicy for storage purposes
* Adjust data format in CustomDate to suit storage purposes
---
* Edit EditCommandTest test cases such that when we are changing the client details, the policies are still kept behind and untouched
* Edit several cases to fix "quarterly" mistake in Frequency to "weekly"
---
* Create JsonAdaptedPolicyTest
* Added some test-cases in JsonAdaptedClientTest

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -65,23 +66,6 @@ public EditCommand(Index index, EditClientDescriptor editClientDescriptor) {
         this.editClientDescriptor = new EditClientDescriptor(editClientDescriptor);
     }
 
-    /**
-     * Creates and returns a {@code Client} with the details of {@code clientToEdit}
-     * edited with {@code editClientDescriptor}.
-     */
-    private static Client createEditedClient(Client clientToEdit, EditClientDescriptor editClientDescriptor) {
-        assert clientToEdit != null;
```
Comment (tituswe) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:73]: Consider changing this to `requireNonNull`?

File: src/main/java/seedu/address/storage/JsonAdaptedPolicy.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        policy.append(policyName);
+        policy.append(", ");
+        policy.append(startDate);
+        policy.append(", ");
+        policy.append(premium);
+        policy.append(", ");
+        policy.append(frequency);
+        /* how to better add tags here ?"
+        policy.append("]");
+        return policy.toString();
```
Comment (tituswe) [inline src/main/java/seedu/address/storage/JsonAdaptedPolicy.java:73]: Will this code be used in the future?

**Review by tituswe — APPROVED** [review]

File: src/main/java/seedu/address/storage/JsonAdaptedPolicy.java (hunk 2 above)
Comment (pangrwa) [reply src/main/java/seedu/address/storage/JsonAdaptedPolicy.java:73]: Might consider in the future, will update!

**Review by tituswe — APPROVED** [review]
LGTM!

**Pull request merged**